ENTERPRISE-MIB

```diff
@@ -0,0 +1,151 @@
+CLEARWATER-ALARMS-MIB DEFINITIONS ::= BEGIN
```

Comment:
Do we want to change the name of the MIB here or the name of the file? My feeling is to change the name of the MIB so that people can tell that this is the Enterprise MIB. Otherwise we'll have two MIB files with similar names: PROJECT-CLEARWATER-MIB &  CLEARWATER-ALARMS-MIB.